crates/sui-pg-db/src/lib.rs

```diff
@@ -12,12 +15,20 @@ use diesel_async::{
     },
     AsyncPgConnection, RunQueryDsl,
 };
-use std::time::Duration;
+use diesel_migrations::{embed_migrations, EmbeddedMigrations};
+use sui_indexer_alt_framework_store_traits::Store;
 use tracing::info;
 use url::Url;
 
+pub use sui_field_count::FieldCount;
```

Comment:
Ah I see -- I think the changes related to `FieldCount` imports that I highlighted earlier came from this. We are now re-exporting `FieldCount` from both here and the top level of the framework. Let's keep it to just the top-level for now, to avoid confusing `rust-analyzer` about where it can pick this import up from, until we decide the best way to generalise it.